editor/docks/groups_editor.h

```diff
@@ -122,6 +124,11 @@ class GroupsEditor : public VBoxContainer {
 
 	void _node_removed(Node *p_node);
 
+	void _add_to_group(const StringName &name, bool persist, const Array &nodes);
+	void _remove_from_group(const StringName &name, const Array &nodes);
+	void _get_group_mask(const StringName &name, Array &nodes, bool invert);
```

Comment:
```suggestion
	void _get_group_mask(const StringName &p_name, Array &r_nodes, bool p_invert);
```
Return arguments should use `r_`.